library/components/extensions-picker/extension-search-bar.tsx

```diff
@@ -39,3 +40,40 @@ export function ExtensionSearchBar(props: ExtensionSearchBarProps) {
     </div>
   );
 }
+
+export function SearchResultsInfo(props: { filter: string; setFilter: React.Dispatch<SetStateAction<string>>, result: ProcessResult }) {
+  function switchOrigin() {
+    if (props.result.originsAny) {
+      props.setFilter(props.filter.replace(ORIGINS_ANY_REGEX, ''));
+    } else {
+      if (props.filter.length == 0) {
+        props.setFilter('origins: any;')
+      } else {
+        props.setFilter(props.filter.replace(/;\s*$/, '') + '; origins:any;');
+      }
+    }
+  }
+
+  function clearFilter() {
+    props.setFilter('');
+  }
+  return (
+    <>
+      {!!props.filter && (
+        <div className="extension-search-clear">
+          {props.result.entries.length > 0 ? <><b>{props.result.entries.length} search results  </b></> : <b>No results found </b>}
+          {props.result.otherOriginsCount > 0 && (
+            <> 
+              <span className='switch-origins'
+                onClick={switchOrigin}>({props.result.originsAny ?
+                  'mask the' : 'show'} {props.result.otherOriginsCount} {props.result.originsAny ? '' : 'other '}results from alternative origins
```

Comment:
```suggestion
                  'hide' : 'show'} {props.result.otherOriginsCount} {props.result.originsAny ? '' : 'other '}results from alternative origins
```

Mask is not a good term to use. Opposite of "show" is "hide".

Reply:
ok then `hide the`